resource-management/test/resourceRegionMgrSimulator/main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 	flag.IntVar(&c.NodesPerRP, "nodes_per_rp", 25000, "The number of RPs per region, if not set, default to 25000")
```

Comment:
Need add line after line 39:

flag.IntVar(&c.RPDownNumber, "rp_down_number", 1, "The number of RPs to be pull down, needs to be <= rp_num")

The above line in PR190 has been merged into branch simulator-list-watch-merge. If above line was not added into this PR and this PR was merged, this line would be removed from the branch simulator-list-watch-merge.

Reply:
this line already included in Ying's PR#190, we don't need add again.
Just resolved all conflict and this PR can be merged now.